src/CodeCoverageExtension.php

```diff
@@ -64,15 +68,13 @@ public function load(ServiceContainer $container, array $params = []): void
 
             if (!isset($options['format'])) {
                 $options['format'] = ['html'];
-            } elseif (!\is_array($options['format'])) {
+            } elseif (!is_array($options['format'])) {
                 $options['format'] = (array) $options['format'];
             }
 
-            if (isset($options['output'])) {
-                if (!\is_array($options['output']) && 1 === \count($options['format'])) {
-                    $format = $options['format'][0];
-                    $options['output'] = [$format => $options['output']];
-                }
+            if (isset($options['output']) && !is_array($options['output']) && 1 === count($options['format'])) {
```

Comment:
This is an unrelated changes and should be reverted.

Reply:
Sorry, it's a miss while I was struggling with fixing test. I agree with atomic commit